pages/performance-management/index.md

```diff
@@ -13,6 +13,7 @@ tags:
   - review
 questions:
   - people-ops
+cSpell: ignore quicklinks
```

Comment:
"quicklinks" is also ignored in multiple pages, add it to `cSpell.json`?

Reply:
✅ Done